packages/frontend/src/features/sqlRunner/components/Header/HeaderCreate.tsx

```diff
@@ -50,7 +54,8 @@ export const HeaderCreate: FC = () => {
 
     const isGithubIntegrationEnabled =
         useFeatureFlagEnabled(FeatureFlags.CustomSQLEnabled) &&
-        health?.data?.hasGithub;
+        health?.data?.hasGithub &&
+        project?.dbtConnection.type === DbtProjectType.GITHUB;
```

Comment:
I believe this boolean should be kept as is, because if someone has a github connection type but github app/bot hasn't been set up yet, then we show the tooltip `Please enable Github integration to write back to dbt in Settings > Integrations > Github'`

Instead, I would create a separate var like `canWriteBackToDbt = project?.dbtConnection.type === DbtProjectType.GITHUB`

and then if that's not true, we don't render the option

What do you think?

Reply:
This `isGithubIntegrationEnabled` flag is what controls the `render` of the `menu item`

So. if doesn't have github ENV variables (hasGithub), or the feature flag is Flase,  or the project is not  on GITHUB, it will not show the menu item, which is likely a thing you can't change easily. 

To check if you have a github integration (aka bot) , we use ` const { isError: githubIsNotInstalled } = useGitHubRepositories();`  If this is false (and `isGithubIntegrationEnabled` is true) , we show the `Please enable Github integration to write back to dbt in Settings > Integrations > Github'`

Reply:
ahhhhh `hasGithub` checks an env that isn't affected by the installation - sorry for adding confusion to this. 

Makes sense the explanation, approving!

Reply:
And tested all scenarios